version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.3"
+version in ThisBuild := "0.2.3-graphql"
```

Comment:
In order to get sbt to resolve the new build configuration appropriately, it's critical that the version is not in the local ivy cache. I've appended this suffix in order to get it to resolve properly. In the future, I'll look into something like `sbt-release` to automate the version bumping / etc. (and including a `-dev` suffix or some-such).